course_catalog/etl/youtube.py

```diff
@@ -101,3 +102,41 @@ def extract():
             yield from get_videos_for_channel(
                 youtube_client, channel["channel_id"], channel["offered_by"]
             )
+
+
+def transform_single_video(offered_by, raw_video_data):
+    """
+    Transforms raw video data into normalized data structure for single video
+
+    Args:
+        offered_by (string)
+        raw_video_data (dict)
+
+    Returns:
+        dict: normalized video data
+    """
+    return {
+        "video_id": raw_video_data["id"],
+        "platform": offered_by,
```

Comment:
This should be `PlatformType.youtube.value`. For `offered_by`, it needs to be a list of dicts, look at xPro as an example.